rootex/framework/systems/transform_system.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include "framework/scene.h"
+#include "framework/system.h"
+#include "components/game/spring_arm_component.h"
+#include "components/game/player_controller.h"
```

Comment:
We shouldn't be including any other components here. This is a transform system which only deals with transform components.